### ISSUE TITLE
Custom emoji in profile editor

### ISSUE THREAD

**Issue by SnowCait** [SnowCait/nostter#1308]

**Comment by dwarf-king-hreidmar** [comment]
Is this going to use NIP-30 ? If so can we also get it in notes? I'm super excited.

Thanks for all the hard work. Nostter is hands down the most solid and bug free nostr client I've used.

**Comment by SnowCait** [comment]
Yes, it is. You can use NIP-30 emoji in notes after bookmarking at https://emojito.meme/.
